meta-luneos/recipes-support/pidgin/icyque_git.bb

```diff
@@ -0,0 +1,25 @@
+SUMMARY = "WIM Protocol plugin for ICQ for Adium, Pidgin, Miranda and Telepathy IM Framework"
+SECTION = "webos/services"
+LICENSE = "GPLv3.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=1ebbd3e34237af26da5dc08a4e440464"
+
+DEPENDS = "pidgin json-glib"
+
+inherit pkgconfig
+
+SRC_URI = "git://github.com/EionRobb/icyque;branch=master;protocol=git"
```

Comment:
set PV

Reply:
And please drop the default ";branch=master;protocol=git"

Reply:
Done & done. Was copied from another recipe. Will clean those up too in another PR.